sms_statistics_service/src/firestore_client.py

```diff
@@ -0,0 +1,27 @@
+import firebase_admin
+from firebase_admin import credentials, firestore
+
+from src.data_models.active_project import ActiveProject
+
+
+class FirestoreClient(object):
```

Comment:
I think this needs a different name - mainly because 'Client' is very heavily used by the Google API libraries.

Consider FirestoreWrapper

Reply:
changed